build.sbt

```diff
@@ -24,7 +22,8 @@ Test / scroogeThriftSourceFolder := { baseDirectory {
 Test / scroogeThriftOutputFolder := (Test / sourceManaged).value
 Test / managedSourceDirectories += (Test / scroogeThriftOutputFolder).value
 
-scalacOptions++= Seq("-deprecation", "-unchecked", "-release:11")
+scalacOptions := Seq("-deprecation", "-release:11")
+javacOptions ++= Seq("--release", "11")
```

Comment:
This `--release` option (a replacement for `-target`, `-source` _and_ the bootclasspath) is new with Java 9, but as we're building with Java 11 or 17, that should be fine!

https://www.baeldung.com/java-compiler-release-option